overlord/devicestate/devicemgr.go

```diff
@@ -939,6 +945,19 @@ func (m *DeviceManager) ensureInstalled() error {
 	}
 	prev = setupRunSystem
 
+	// add the install-device hook before ensure-next-boot-to-run-mode if it
+	// exists in the snap
+	if hasInstallDeviceHook {
+		summary := i18n.G("Run install-device hook")
+		hooksup := &hookstate.HookSetup{
+			// TODO: what's a reasonable timeout for the install-device hook?
```

Comment:
the default is 10 mins nowadays (I thought it was shorter but was wrong), which seems fair not having any example yet